### PR TITLE
Add Database landing page

### PR DESCRIPTION
**Pull request by rachelwhitton**
@nataliejeremy can you review and merge?

### CODE REVIEW

**Comment by rachelwhitton** [comment]
Closes #505 and contributes to #423

File: source/docs/articles/sites/database.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 ---
 title: MySQL Database
 description: Learn about the Database that runs in your site
-category:
-- developing
-draft: true
+keywords: mysql, sql, database, db, databases
 ---
+The Pantheon platform provides each site a dedicated MySQL container, which can be maintained remotely or locally depending on your workflow. For a comprehensive list of MySQL settings [access your database](/docs/articles/local/accessing-mysql-databases/#database-connection-information) and use the [SHOW VARIABLES](http://dev.mysql.com/doc/refman/5.0/en/show-variables.html) statement.
```
Comment (bmackinney) [inline source/docs/articles/sites/database.md]: running mariadb.

Comment (bmackinney) [inline source/docs/articles/sites/database.md]: each site environment

File: source/docs/articles/sites/database.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+
+## Maintain Database Content and Configurations
+The [Pantheon Workflow](/docs/articles/sites/code/using-the-pantheon-workflow/) moves content down from your Live environment to Test and then to Dev. This means that any content added via the database to your Live site can be imported into the existing database on the Test and Dev environments using the Workflow tool.
+
+You can use [PHPMyAdmin](/docs/articles/local/using-phpmyadmin-to-locally-administer-a-database/) or [MySQL Workbench](/docs/articles/local/using-mysql-workbench-to-access-a-database/) to  administer your site's database
+and [export configurations](/docs/articles/sites/code/using-the-pantheon-workflow/#exporting-configuration) as needed.
```
Comment (bmackinney) [inline source/docs/articles/sites/database.md]: PHPMyAdmin and MySQL workbench aren't the tools we recommend in that part of the article. Split this part into its own section, warning about overwriting configurations on a db clone  task.

File: source/docs/articles/sites/database.md [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
 description: Learn about the Database that runs in your site
-category:
-- developing
-draft: true
+keywords: mysql, sql, database, db, databases
 ---
+The Pantheon platform provides each site a dedicated MySQL container, which can be maintained remotely or locally depending on your workflow. For a comprehensive list of MySQL settings [access your database](/docs/articles/local/accessing-mysql-databases/#database-connection-information) and use the [SHOW VARIABLES](http://dev.mysql.com/doc/refman/5.0/en/show-variables.html) statement.
+
+## Maintain Database Content and Configurations
+The [Pantheon Workflow](/docs/articles/sites/code/using-the-pantheon-workflow/) moves content down from your Live environment to Test and then to Dev. This means that any content added via the database to your Live site can be imported into the existing database on the Test and Dev environments using the Workflow tool.
```
Comment (bmackinney) [inline source/docs/articles/sites/database.md]: Rephrase. Using the default workflow, you can overwrite your dev or test db's with live's. In the second sentence, clarify not just content comes.

File: source/docs/articles/sites/database.md (hunk 1 above)
Comment (rachelwhitton) [inline source/docs/articles/sites/database.md]: @bmackinney can you explain more about "running mariadb" or provide a link you'd like to see included for more info/context? I'm not familiar with it.

**Comment by rachelwhitton** [comment]
@bmackinney can you review? 
The only suggestion I did not work in was the one to add "running mariadb"

Added image for workflow tool - image has already been optimized/compressed.

**Pull request merged**